codalab_service.py

```diff
@@ -353,6 +353,11 @@ def has_callable_default(self):
         help='Comma-separated list of directories that are mounted on the REST server, allowing their contents to be used in the --link argument.',
         default='/tmp/codalab/link-mounts',
     ),
+    CodalabArg(name='azure_blob_connection_string', help='Azure Blob storage connection string'),
+    CodalabArg(
```

Comment:
Why do we need this new argument?

Reply:
I use it in GitHub Actions tests so that everything (including all uploads to / from the worker) use Azure Blob Storage. I'll add a comment that mentions it's only used for tests.